components/org.wso2.carbon.identity.conditional.auth.functions.choreo/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/choreo/CallChoreoFunctionImpl.java

```diff
@@ -266,7 +278,7 @@ private void requestAccessToken(String tenantDomain, AccessTokenRequestHelper ac
 
         request.setHeader(AUTHORIZATION, BASIC + Base64.getEncoder()
                 .encodeToString((accessTokenRequestHelper.consumerKey + ":" + accessTokenRequestHelper.consumerSecret)
-                .getBytes(StandardCharsets.UTF_8)));
+                        .getBytes(StandardCharsets.UTF_8)));
```

Comment:
is the formatting correct?

Reply:
Fixed by https://github.com/wso2-extensions/identity-conditional-auth-functions/pull/146/commits/8f9c5ceaa6df55aaed1ed4500bbcd7192ac903c7